specification/imagebuilder/resource-manager/readme.md

```diff
@@ -133,3 +142,11 @@ See configuration in [readme.go.md](./readme.go.md)
 ## Java
 
 See configuration in [readme.java.md](./readme.java.md)
+
+## Suppression
+
+``` yaml
+directive:
+  - suppress: AvoidAdditionalProperties
```

Comment:
>AvoidAdditionalProperties [](http://example.com/codeflow?start=14&length=25)

Invalid syntax, you should also add the where propery

Reply:
Here is a readme file as an example:  https://github.com/Azure/azure-rest-api-specs-pr/blob/8362df48a86778d5645e6d56b2dccdd26f223353/specification/maintenance/resource-manager/readme.md